shared/src/business/entities/EntityConstants.ts

```diff
@@ -415,6 +415,12 @@ export const EVENT_CODES_REQUIRING_JUDGE_SIGNATURE = without(
   'NOT',
 );
 
+const excludedOrderEventCodes = ['OAJ', 'SPOS', 'SPTO', 'OST'];
```

Comment:
I think the name excludedOrderEventCodes becomes confusing in this context of EntityConstants. A rename might be in order, or just removed and the codes moved into the `JUDGE_ACTIVITY_REPORT_ORDER_EVENT_CODES` filter function

Reply:
Good point!

Reply:
Ok.. Resolved. [c6ac968](https://github.com/ustaxcourt/ef-cms/pull/3983/commits/c6ac9681ca91d6986ca224cd7ca0148f53c2fc24)